src/peer_connection/mod.rs

```diff
@@ -1217,7 +1239,43 @@ impl RTCPeerConnection {
 
         let we_answer = desc.sdp_type == RTCSdpType::Answer;
         let remote_description = self.remote_description().await;
+        let mut local_transceivers = self.get_transceivers().await;
         if we_answer {
+            if let Some(parsed) = desc.parsed {
+                // WebRTC Spec 1.0 https://www.w3.org/TR/webrtc/
+                // Section 4.4.1.5
+                for media in &parsed.media_descriptions {
+                    let mid_value = match get_mid_value(media) {
+                        Some(mid) => mid,
```

Comment:
It will fail silently later(`find_by_mid` will return None). I'm not quite sure if this should be an error here, but for now silent failure seems good enough.